enlighten/assets/uic_qrc/enlighten_layout.ui

```diff
@@ -14150,6 +14157,182 @@ p, li { white-space: pre-wrap; }
+                          <size>
+                           <width>75</width>
+                           <height>0</height>
+                          </size>
+                         </property>
+                         <property name="alignment">
+                          <set>Qt::AlignCenter</set>
+                         </property>
+                         <property name="minimum">
+                          <double>1.000000000000000</double>
+                         </property>
+                         <property name="maximum">
+                          <double>100.000000000000000</double>
+                         </property>
+                         <property name="singleStep">
+                          <double>0.100000000000000</double>
+                         </property>
+                         <property name="value">
+                          <double>7.000000000000000</double>
+                         </property>
+                        </widget>
+                       </item>
+                       <item row="1" column="1">
+                        <widget class="QLabel" name="label_despike_tau">
+                         <property name="toolTip">
+                          <string>Sensitivty to Peaks. Lower values means flagging more pixels as potential spikes.</string>
+                         </property>
+                         <property name="text">
+                          <string>Tau</string>
+                         </property>
+                        </widget>
+                       </item>
+                       <item row="2" column="0">
+                        <widget class="QSpinBox" name="spinBox_window_despike">
+                         <property name="minimumSize">
+                          <size>
+                           <width>75</width>
+                           <height>0</height>
+                          </size>
+                         </property>
+                         <property name="alignment">
+                          <set>Qt::AlignCenter</set>
+                         </property>
+                         <property name="minimum">
+                          <number>1</number>
+                         </property>
+                         <property name="maximum">
+                          <number>100</number>
+                         </property>
+                         <property name="value">
+                          <number>3</number>
+                         </property>
+                        </widget>
+                       </item>
+                       <item row="2" column="1">
+                        <widget class="QLabel" name="label_despike_window">
+                         <property name="toolTip">
+                          <string>The bounds for the area of pixels averaged in a spike. This is for one side. A value of 3 means a window of 6, 2 would be a window of 4 pixels etc.</string>

```

Comment:
These tooltips are great, I just think it would be good to copy them to the code as well — programmers may not think to look in the GUI schema to find them.